### PR TITLE
Update Jenkinsfile

### PR DESCRIPTION
**Pull request by fonsecas72**

### CODE REVIEW

**Comment by dmr0689** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">Please include a CHANGELOG entry. <br />
You can find it at <a href="https://github.com/realm/jazzy/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true"><a href="https://github.com/beubi/dummy/blob/a539069dc57a26b130307fe259cfc98dd254cc09/Jenkinsfile #L3">Jenkinsfile #L3</a> - SHIT</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Message … .. .  .safaf</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Note, we hard-wrap at 80 chars and use 2 spaces after the last line.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger_id">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: Jenkinsfile [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 stage 'Trying'
```
Comment (CapitaoCuecas) [inline Jenkinsfile]: <table data-meta="generated_by_danger_id">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="true"><span data-href="https://github.com/beubi/dummy/blob/a539069dc57a26b130307fe259cfc98dd254cc09/Jenkinsfile#L3"></span>SHIT</td>
    </tr>
  </tbody>
</table>

**Comment by dmr0689** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">Please include a CHANGELOG entry. <br />
You can find it at <a href="https://github.com/realm/jazzy/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Message … .. .  .safaf</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Note, we hard-wrap at 80 chars and use 2 spaces after the last line.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger_id">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: Jenkinsfile (hunk 1 above)
Comment (dmr0689) [inline Jenkinsfile:3]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="true"><span data-href="https://github.com/beubi/dummy/blob/a539069dc57a26b130307fe259cfc98dd254cc09/Jenkinsfile#L3"></span>SHIT</td>
    </tr>
  </tbody>
</table>

**Comment by dmr0689** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">Please include a CHANGELOG entry. <br />
You can find it at <a href="https://github.com/realm/jazzy/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Message … .. .  .safaf</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Note, we hard-wrap at 80 chars and use 2 spaces after the last line.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Pull request closed without merging**